openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/DomainFilterQueryFilter.spec.ts

```diff
@@ -0,0 +1,999 @@
+/*
+ *  Copyright 2025 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import base, { expect, Page } from '@playwright/test';
+import { get } from 'lodash';
+import { SidebarItem } from '../../constant/sidebar';
+import { Domain } from '../../support/domain/Domain';
+import { SubDomain } from '../../support/domain/SubDomain';
+import { TableClass } from '../../support/entity/TableClass';
+import { TopicClass } from '../../support/entity/TopicClass';
+import { performAdminLogin } from '../../utils/admin';
+import {
+  getApiContext,
+  redirectToExplorePage,
+  redirectToHomePage,
+} from '../../utils/common';
+import {
+  checkAssetsCount,
+  selectDomain,
+  verifyActiveDomainIsDefault,
+} from '../../utils/domain';
+import { waitForAllLoadersToDisappear } from '../../utils/entity';
+import { sidebarClick } from '../../utils/sidebar';
+
+const test = base.extend<{ page: Page }>({
+  page: async ({ browser }, use) => {
+    const { page } = await performAdminLogin(browser);
+    await use(page);
+    await page.close();
+  },
+});
+
+test.describe('Domain Filter - User Behavior Tests', () => {
```

Comment:
Add one more test with combination of explore quick filters. Create 3 tables, assign 2 tables to domain. Apply tier/tag to all three tables. 
First apply tier/tag quick filter, verify all 3 tables to be visible.
Then apply domain selection from navbar, verify 2 tables to appear. 
Clear domain selection and quick filter should persist with 3 tables visible.